debian/tree/zfsutils-linux/usr/lib/zfs-linux/zvol-wait

```diff
@@ -0,0 +1,84 @@
+#!/bin/sh
```

Comment:
I'd suggest moving this script in to the `etc/systemd/system/` directory, or a new `etc/systemd/scripts` directory in the git tree.  The right install location for this on the system appears to be `/usr/lib/systemd/scripts/` (checked on CentOS 7 and Ubuntu 18.04), but it should also be possible to override it with `./configure`.  You'll  need to add a `systemdscriptsdir` to `config/user-systemd.m4` like the others for this.

Reply:
Looking at other systemd ExecStart scripts, most of the system ones are placed into `/lib/systemd/` directly and are named `systemd-something`.

A bunch of other scripts are placed all around the place, with many in `/bin/`, `/usr/bin/`, `/sbin`, etc.

Thinking about it a bit more, it might be convenient to have this function available for manual execution, so having it in `/bin` or `/usr/bin` seems to make more sense to me. We could rename it to something like `zfs-volume-wait`.

Reply:
I wasn't able to dig up the documentation to confirm this, but it looks to me like everything in `/lib/systemd/` in an arch specific binary, and `/lib/systemd/scripts/` is for portable arch-independent scripts.

But I agree, if we want to be able to easily use this utility outside of systemd then it really belongs in `/usr/bin/`.  In which case I'd suggest putting the source in a subdirectory under ./cmd/, which is where we put things like `zvol_id`.